src/main/java/Duke.java

```diff
@@ -50,22 +50,58 @@ public static void listActions() {
                 break;
             }else if(line.equalsIgnoreCase("list")) {
```

Comment:
for the if else loop. You can follow this
if (condition) {
statements;
} else if (condition) {
statements;
} else {
statements;
}
Leave a space between else/if and bracket.
And also It is not suggested to use so many if else if.